### ISSUE TITLE
Tick points sometimes errors

### ISSUE THREAD

**Issue by nerrage** [lucas-r-cooper/Coreuptedbot#7]
Seems that sometimes the API doesn't reply nicely, gotta add some protections around points.

**Comment by nerrage** [comment]
```>>> import core_functions
>>> core_functions.getchatlist()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "core_functions.py", line 22, in getchatlist
    bad_hombres = json.loads(r.text)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
```

**Comment by nerrage** [comment]
Inconsistent though :(

**Comment by nerrage** [comment]
Fixed in f7fee0e095f2399bfac161958245a48c5e2886b7
